### PR TITLE
making language consistent with emails, see #55

### PR DESCRIPTION
**Pull request by wcjr**

### CODE REVIEW

File: dictionaries/login.definition.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
 {
 	"title": {
-		"en": "Login with your {idpName} account",
-		"es": "Inicie sesión con su cuenta {idpName}",
-		"fr": "Connectez-vous avec votre compte {idpName}",
-		"ko": "{idpName} 계정으로 로그인하십시오."
+		"en": "Login with your {idpName} identity",
+		"es": "Inicia sesión con tu identidad de {idpName}",
```
Comment (forevermatt) [inline dictionaries/login.definition.json]: Might want to double-check this Spanish translation with someone. It looks like it switch from the formal (su) to the informal (tu). I'm not sure about the other changes.

File: dictionaries/login.definition.json [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+		"fr": "Connectez-vous avec votre identité {idpName}",
+		"ko": "{idpName} 신원 계정으로 로그인하십시오."
 	},
 	"header": {
-		"en": "Login with your {idpName} account",
-		"es": "Inicie sesión con su cuenta {idpName}",
-		"fr": "Connectez-vous avec votre compte {idpName}",
-		"ko": "{idpName} 계정으로 로그인하십시오."
+		"en": "Login with your {idpName} identity",
+		"es": "Inicia sesión con tu identidad de {idpName}",
```
Comment (forevermatt) [inline dictionaries/login.definition.json]: Same as above

File: dictionaries/selectidp.definition.json [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
 {
 	"title": {
-		"en": "Login account",
-		"es": "Cuenta de inicio de sesión",
-		"fr": "Compte de connexion",
-		"ko": "로그인 계정"
+		"en": "Choose an identity account",
+		"es": "Elige una cuenta de identidad",
+		"fr": "Choisissez un compte d'identité",
+		"ko": "ID 계정 선택"
```
Comment (forevermatt) [inline dictionaries/selectidp.definition.json]: It's interesting that these Korean translations now contain Roman characters ("ID").

File: dictionaries/mfa.definition.json [hunk 4]
```diff
@@ -115,10 +115,10 @@
 		"ko": "자기 보호"
 	},
 	"nag_info": {
-		"en": "Did you know you could easily increase the security of your account by enabling 2-Step Verification?",
-		"es": "¿Sabía que podría aumentar fácilmente la seguridad de su cuenta al habilitar la verificación en dos pasos?",
-		"fr": "Savez-vous que vous pouvez facilement augmenter la sécurité de votre compte en activant la vérification en deux étapes?",
-		"ko": "2 단계 인증을 사용하여 계정의 보안성을 쉽게 높일 수 있다는 사실을 알고 계셨습니까?"
+		"en": "Did you know you could easily increase the security of your identity account by enabling 2-Step Verification?",
+		"es": "¿Sabía que podría aumentar fácilmente la seguridad de su cuenta de identidad al habilitar la verificación en dos pasos?",
+		"fr": "Saviez-vous que vous pouvez facilement augmenter la sécurité de votre compte d'identité en activant la validation en deux étapes?",
```
Comment (Baggerone) [inline dictionaries/mfa.definition.json]: fr ...
"Savez-vous que vous pouvez facilement augmenter la sécurité de votre compte d'identité en activant la vérification en deux étapes?",

File: dictionaries/login.definition.json (hunk 1 above)
Comment (wcjr) [reply dictionaries/login.definition.json]: Thanks, I'll keep it close to what was there before

File: dictionaries/login.definition.json (hunk 2 above)
Comment (wcjr) [reply dictionaries/login.definition.json]: ditto

File: dictionaries/selectidp.definition.json (hunk 3 above)
Comment (wcjr) [reply dictionaries/selectidp.definition.json]: I thought so too, I'll ask about that one.

File: dictionaries/mfa.definition.json [hunk 5]
```diff
@@ -127,10 +127,10 @@
 		"ko": "이 계정 보호"
 	},
 	"required_info": {
-		"en": "Your account requires additional security, you must set up 2-Step Verification at this time.",
-		"es": "Su cuenta requiere seguridad adicional, debe configurar la verificación en dos pasos en este momento",
-		"fr": "Votre compte nécessite une sécurité supplémentaire, vous devez configurer la vérification en deux étapes en ce moment",
-		"ko": "계정에 대한 추가 보안이 필요하므로 지금 2 단계 인증을 설정해야합니다."
+		"en": "Your identity account requires additional security, you must set up 2-Step Verification at this time.",
+		"es": "Su cuenta de identidad requiere seguridad adicional, debe configurar la verificación en dos pasos en este momento.",
+		"fr": "Votre compte d'identité nécessite une sécurité supplémentaire, vous devez configurer la validation en deux étapes pour le moment.",
```
Comment (Baggerone) [inline dictionaries/mfa.definition.json]: fr ...
"Votre compte d'identité nécessite une sécurité supplémentaire, vous devez configurer la vérification en deux étapes en ce moment"

File: dictionaries/mfa.definition.json [hunk 6]
```diff
@@ -181,10 +181,10 @@
 		"ko": "새로운 인쇄 가능한 코드를 만드는 동안 문제가 발생했습니다. 불편을 끼쳐 드려 죄송합니다. 계속 진행 한 후 다음 주소로 구성을 확인하십시오. "
 	},
 	"account": {
-		"en": "{idpName} account",
-		"es": "Cuenta de {idpName}",
-		"fr": "Compte {idpName}",
-		"ko": "{idpName} 계정"
+		"en": "{idpName} identity account",
+		"es": "Cuenta de identidad de {idpName}",
+		"fr": "{idpName} compte d'identité",
```
Comment (Baggerone) [inline dictionaries/mfa.definition.json]: fr ...
"compte d'identité {idpName} "

File: dictionaries/mfa.definition.json (hunk 4 above)
Comment (wcjr) [reply dictionaries/mfa.definition.json]: got it, thanks

File: dictionaries/mfa.definition.json (hunk 5 above)
Comment (wcjr) [reply dictionaries/mfa.definition.json]: got it, thanks

File: dictionaries/mfa.definition.json (hunk 6 above)
Comment (wcjr) [reply dictionaries/mfa.definition.json]: got it, thanks

**Pull request merged**